Kconfig.zephyr

```diff
@@ -360,9 +360,13 @@ config NO_OPTIMIZATIONS
 	help
 	  Compiler optimizations will be set to -O0 independently of other
 	  options.
-
 endchoice
 
+config ENABLE_WARNING_AS_ERRORS
+	bool "Stop building if warnings occures"
```

Comment:
Wouldn't "Treat warnings as errors" be more adequate name for this Kconfig? IMHO, its effect ("Stop building if warnings occures") should be part of the description/help.

Reply:
Thanks for comment! I agree